specification/containerinstance/resource-manager/Microsoft.ContainerInstance/preview/2024-11-01-preview/containerInstance.json

```diff
@@ -0,0 +1,2663 @@
+                "type"
+              ]
+            }
+          }
+        },
+        "volumeMounts": {
+          "description": "The volume mounts available to the init container.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/VolumeMount"
+          },
+          "x-ms-identifiers": [
+            "name"
+          ]
+        },
+        "securityContext": {
+          "description": "The container security properties.",
+          "$ref": "#/definitions/SecurityContextDefinition"
+        }
+      }
+    },
+    "CloudError": {
+      "x-ms-external": true,
+      "properties": {
+        "error": {
+          "$ref": "#/definitions/CloudErrorBody"
+        }
+      },
+      "description": "An error response from the Container Instance service."
+    },
+    "CloudErrorBody": {
+      "x-ms-external": true,
+      "properties": {
+        "code": {
+          "type": "string",
+          "description": "An identifier for the error. Codes are invariant and are intended to be consumed programmatically."
+        },
+        "message": {
+          "type": "string",
+          "description": "A message describing the error, intended to be suitable for display in a user interface."
+        },
+        "target": {
+          "type": "string",
+          "description": "The target of the particular error. For example, the name of the property in error."
+        },
+        "details": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/CloudErrorBody"
+          },
+          "x-ms-identifiers": [
+            "message",
+            "target"
+          ],
+          "description": "A list of additional details about the error."
+        }
+      },
+      "description": "An error response from the Container Instance service."
+    },
+    "NetworkDependenciesResponse": {
+      "description": "Response for network dependencies, always empty list.",
+      "type": "array",
+      "items": {
+        "type": "string"
+      }
+    },
+    "ConfidentialComputeProperties": {
+      "description": "The properties for confidential container group",
+      "type": "object",
+      "properties": {
+        "ccePolicy": {
+          "type": "string",
+          "description": "The base64 encoded confidential compute enforcement policy"
+        }
+      }
+    },
+    "IdentityAcls": {
+      "description": "The access control levels of the identities.",
+      "type": "object",
+      "properties": {
+        "defaultAccess": {
+          "description": "The default access level.",
+          "type": "string",
+          "enum": [
+            "All",
+            "System",
+            "User"
+          ],
+          "x-ms-enum": {
+            "name": "IdentityAccessLevel",
+            "modelAsString": true
+          }
+        },
+        "acls": {
+          "description": "The access control levels for each identity.",
+          "type": "array",
+          "items": {
```

Comment:
plese check the swagger lint diff warning for this property : 

[https://github.com/Azure/azure-rest-api-specs/pull/30879/checks?check_run_id=31300522952](https://github.com/Azure/azure-rest-api-specs/pull/30879/checks?check_run_id=31300522952)

[https://github.com/Azure/azure-openapi-validator/blob/main/docs/xms-identifier-validation.md](https://github.com/Azure/azure-openapi-validator/blob/main/docs/xms-identifier-validation.md)

Reply:
Thank you, I fixed that issue, please let me know if there is anything else to fix.

Reply:
Hello @raosuhas , is there any update on this PR? Thanks!